<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>目标检测系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #f5f5f5;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>6</number>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="leftPanel" native="true">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="leftPanelLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QGroupBox" name="displayModeGroup">
          <property name="title">
           <string>显示模式</string>
          </property>
          <layout class="QVBoxLayout" name="displayModeLayout">
           <property name="spacing">
            <number>8</number>
           </property>
           <item>
            <widget class="QRadioButton" name="originalRadio">
             <property name="text">
              <string>原始内容</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="detectionRadio">
             <property name="text">
              <string>识别结果</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="binaryRadio">
             <property name="text">
              <string>二值化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="roiRadio">
             <property name="text">
              <string>ROI切割</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="mediaInfoGroup">
          <property name="title">
           <string>媒体信息</string>
          </property>
          <layout class="QVBoxLayout" name="mediaInfoLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="mediaTypeLabel">
             <property name="text">
              <string>类型：未加载</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mediaSizeLabel">
             <property name="text">
              <string>分辨率：-</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mediaInfoLabel">
             <property name="text">
              <string>详细信息：-</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="paramGroup">
          <property name="title">
           <string>参数设置</string>
          </property>
          <layout class="QVBoxLayout" name="paramLayout">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="confidenceHeaderLayout">
             <item>
              <widget class="QLabel" name="confidenceLabel">
               <property name="text">
                <string>置信度：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="confidenceValueLabel">
               <property name="styleSheet">
                <string notr="true">color: #0078d4; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0.50</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="confidenceSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="confidenceSlider">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>50</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="nmsHeaderLayout">
             <item>
              <widget class="QLabel" name="nmsLabel">
               <property name="text">
                <string>NMS 阈值：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="nmsValueLabel">
               <property name="styleSheet">
                <string notr="true">color: #0078d4; font-weight: bold;</string>
               </property>
               <property name="text">
                <string>0.40</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="nmsSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QSlider" name="nmsSlider">
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>40</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="roiHeaderLayout">
             <item>
              <widget class="QLabel" name="roiLabel">
               <property name="text">
                <string>ROI 大小：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="roiSizeSpinBox">
               <property name="suffix">
                <string> px</string>
               </property>
               <property name="minimum">
                <number>100</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="singleStep">
                <number>10</number>
               </property>
               <property name="value">
                <number>640</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="detectionGroup">
          <property name="title">
           <string>检测结果</string>
          </property>
          <layout class="QVBoxLayout" name="detectionLayout">
           <item>
            <widget class="QListWidget" name="detectionList">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="leftPanelSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="displayLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>300</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottomPanel" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="bottomPanelLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QSlider" name="progressSlider">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="playbackInfoLayout">
         <item>
          <widget class="QLabel" name="frameLabel">
           <property name="text">
            <string>帧：0/0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fpsLabel">
           <property name="text">
            <string>FPS：0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="detectionCountLabel">
           <property name="text">
            <string>检测：0 个目标</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="playbackInfoSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenVideo"/>
    <addaction name="separator"/>
    <addaction name="actionSaveFrame"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuTheme">
     <property name="title">
      <string>主题(&amp;T)</string>
     </property>
     <action name="actionLightTheme">
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>浅色主题(&amp;L)</string>
      </property>
      <property name="statusTip">
       <string>切换到浅色主题</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+L</string>
      </property>
     </action>
     <action name="actionDarkTheme">
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>深色主题(&amp;D)</string>
      </property>
      <property name="statusTip">
       <string>切换到深色主题</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Shift+D</string>
      </property>
     </action>
     <addaction name="actionLightTheme"/>
     <addaction name="actionDarkTheme"/>
    </widget>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionFitWindow"/>
    <addaction name="actionActualSize"/>
    <addaction name="separator"/>
    <addaction name="actionToggleLeftPanel"/>
    <addaction name="menuTheme"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionLoadModel"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>主工具栏</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenImage"/>
   <addaction name="actionOpenVideo"/>
   <addaction name="separator"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionSaveFrame"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpenImage">
   <property name="icon">
    <iconset theme=":/icons/image">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>打开图片(&amp;I)...</string>
   </property>
   <property name="statusTip">
    <string>打开图片文件进行识别</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="icon">
    <iconset theme=":/icons/video">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>打开视频(&amp;O)...</string>
   </property>
   <property name="statusTip">
    <string>打开视频文件进行识别</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveFrame">
   <property name="icon">
    <iconset theme=":/icons/save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>保存当前帧(&amp;S)...</string>
   </property>
   <property name="statusTip">
    <string>保存当前显示的帧到文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset theme=":/icons/export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>导出结果(&amp;E)...</string>
   </property>
   <property name="statusTip">
    <string>导出检测结果到文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>退出应用程序</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="statusTip">
    <string>撤销上一步操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>重做上一步操作</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>放大(&amp;I)</string>
   </property>
   <property name="statusTip">
    <string>放大显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>缩小(&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>缩小显示</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionFitWindow">
   <property name="text">
    <string>适应窗口(&amp;F)</string>
   </property>
   <property name="statusTip">
    <string>调整图片适应窗口大小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionActualSize">
   <property name="text">
    <string>实际大小(&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>显示图片实际大小</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionToggleLeftPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示/隐藏左侧面板(&amp;L)</string>
   </property>
   <property name="statusTip">
    <string>切换左侧控制面板的显示状态</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="text">
    <string>加载模型(&amp;M)...</string>
   </property>
   <property name="statusTip">
    <string>加载目标检测模型</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>参数设置(&amp;P)...</string>
   </property>
   <property name="statusTip">
    <string>打开参数设置对话框</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)...</string>
   </property>
   <property name="statusTip">
    <string>关于本软件</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset theme=":/icons/play">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="toolTip">
    <string>播放视频</string>
   </property>
   <property name="statusTip">
    <string>开始播放视频</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset theme=":/icons/pause">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="toolTip">
    <string>暂停播放</string>
   </property>
   <property name="statusTip">
    <string>暂停视频播放</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme=":/icons/stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止播放</string>
   </property>
   <property name="statusTip">
    <string>停止视频播放并回到开头</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
